test/modules/linear_boltzmann_solvers/transport_steady/transport_3d_1a_extruder.lua

```diff
@@ -3,46 +3,45 @@
 -- Test: Max-value=5.27450e-01 and 3.76339e-04
 num_procs = 4
 
-
-
-
-
---############################################### Check num_procs
-if (check_num_procs==nil and number_of_processes ~= num_procs) then
-  log.Log(LOG_0ERROR,"Incorrect amount of processors. " ..
-    "Expected "..tostring(num_procs)..
-    ". Pass check_num_procs=false to override if possible.")
+-- ########## Check num_procs ##########
+if (check_num_procs == nil and number_of_processes ~= num_procs) then
+  log.Log(LOG_0ERROR, "Incorrect amount of processors. " .. "Expected " .. tostring(num_procs) ..
+            ". Pass check_num_procs=false to override if possible.")
   os.exit(false)
 end
 
---############################################### Setup mesh
-meshgen1 = mesh.ExtruderMeshGenerator.Create
-({
-  inputs =
-  {
-    mesh.FromFileMeshGenerator.Create
-    ({
+-- ########## Setup mesh ##########
+meshgen1 = mesh.ExtruderMeshGenerator.Create({
+  inputs = {
+    mesh.FromFileMeshGenerator.Create({
       filename = "../../../../resources/TestMeshes/SquareMesh2x2Quads.obj"
-    }),
+    })
   },
-  layers = {{z=0.4,n=2},{z=0.8,n=2},{z=1.2,n=2},{z=1.6,n=2}}, -- layers
-  partitioner = mesh.KBAGraphPartitioner.Create
-  ({
-    nx = 2, ny=2, nz=1,
-    xcuts = {0.0}, ycuts = {0.0}
+  layers = {{z = 0.4, n = 2}, {z = 0.8, n = 2}, {z = 1.2, n = 2}, {z = 1.6, n = 2}}, -- layers
+  partitioner = mesh.KBAGraphPartitioner.Create({
+    nx = 2,
+    ny = 2,
+    nz = 1,
+    xcuts = {0.0},
+    ycuts = {0.0}
   })
 })
 mesh.MeshGenerator.Execute(meshgen1)
 
---############################################### Set Material IDs
-vol0 = logvol.RPPLogicalVolume.Create({infx=true, infy=true, infz=true})
-mesh.SetMaterialIDFromLogicalVolume(vol0,0)
+-- ########## Set Material IDs ##########
+vol0 = logvol.RPPLogicalVolume.Create({infx = true, infy = true, infz = true})
+mesh.SetMaterialIDFromLogicalVolume(vol0, 0)
 
-vol1 = logvol.RPPLogicalVolume.Create
-({ xmin=-0.5,xmax=0.5,ymin=-0.5,ymax=0.5, infz=true })
-mesh.SetMaterialIDFromLogicalVolume(vol1,1)
+vol1 = logvol.RPPLogicalVolume.Create({
+  xmin = -0.5,
+  xmax = 0.5,
+  ymin = -0.5,
+  ymax = 0.5,
+  infz = true
+})
+mesh.SetMaterialIDFromLogicalVolume(vol1, 1)
 
---############################################### Add materials
+-- ########## Add materials ##########
```

Comment:
I think we are of the opinion that, in comments, we no longer want to see #########

Reply:
I've removed them from the imput files.